src/pypwsqc/indicator_correlation.py

```diff
@@ -155,6 +187,12 @@ def ic_filter(
         Indicator correlation threshold below `quantile_bin_ref` where PWS are
         still accepted
 
+    Literature
+    ----------
+    [1] Bárdossy, A., Seidel, J., and El Hachem, A.: The use of personal weather station 
+    observations to improve precipitation estimation and interpolation, 
+    Hydrol. Earth Syst. Sci., 25, 583–601, https://doi.org/10.5194/hess-25-583-2021, 2021. 
```

Comment:
the CI linting error seems to stem from `583–601` which should be `583-601` with a "normal minus sign". Not really important but can be easily fixed.